### DISCUSSION TITLE
Explorers' Leaves

### DISCUSSION THREAD

**Discussion by kitmonkman** [snezhig/obsidian-front-matter-title#203] [Q&A]
Hi,

Your plugin is exactly what I've been looking for. Thank you. The only (quite significant) issue, is that it works on everything EXCEPT the File Explorer. On loading Obsidian, I'm getting a pop-up error "Error: There are some explorers' leaves", though I don't see anything in the Console. Do you have any ideas what might be causing this?

Many thanks

Kit

**Comment by snezhig** [comment]
Hi.
Do you use another plugins?
My plugin checks leaves by using `app.workspace.getLeavesOfType('file-explorer')` and the errors appears if there are more than one leaf.

**Comment by kitmonkman** [comment]
Hi,

Thanks for the swift response, it’s hugely appreciated!

I’ve turned off all other plugins but even with FrontMatter as the only one, I’m still getting the same error message.

Thanks

Kit

**Reply by snezhig** [reply]
You can try to run `app.workspace.getLeavesOfType('file-explorer')` in console and show me the output. Or copy it in a file and send it.

**Comment by kitmonkman** [comment]
Hi,

This is what I’m getting:

￼

V best

Kit

**Reply by snezhig** [reply]
May be you have specific settings in obsidian app?

**Comment by kitmonkman** [comment]
I have a Catalyst license, so I am on version 1.5.3, otherwise I don’t think anything’s specific. I’ll keep experimenting with settings. I’m so looking forward to fixing this.

Thanks again for all your help and for the wonderful plugin. 

V best

Kit

**Reply by snezhig** [reply]
![изображение](https://github.com/snezhig/obsidian-front-matter-title/assets/3987716/dcac486f-147f-46cb-9e7a-1da1a0584ecc)
Can you give me something like this? You should have more than 1 element.

**Comment by kitmonkman** [comment]
Is this more helpful?

￼

**Reply by snezhig** [reply]
Do not copy it, do a screenshot. :)

**Comment by kitmonkman** [comment]
Hi,

Apologies for being slow, but I attached a screenshot. Here’s a link to it, along with an embedded attachment:

https://www.dropbox.com/scl/fi/2tud52r1h5mqkl6lobjvr/Screenshot-2024-01-13-at-12.33.38.jpg?rlkey=sz6rbkblex9joz0ki24z7sg8r&dl=0￼
Screenshot 2024-01-13 at 12.33.38.jpg
dropbox.com


If that’s not what you need, do let me know and I’ll try again.

Many thanks!

Kit

￼

**Comment by snezhig** [comment]
Okay. to step by step. These are "parent" property in two elements. Can you expand them and make a screenshots?

**Comment by kitmonkman** [comment]
Hi,

Attached screenshots (I hope I understood you correctly)

V best

Kit



￼￼

**Reply by snezhig** [reply]
I do not see any. Previous link open the previous screenshot only.

**Comment by kitmonkman** [comment]
Hi,

For some reason, my attachments aren’t getting through to you.

I’ve put those two screenshots on Dropbox in this folder: https://www.dropbox.com/scl/fo/4yfo0tq2sws62x3awmz37/h?rlkey=uc8zyjz1nfb8zovgzk71hu23r&dl=0

Thanks

Kit

**Reply by snezhig** [reply]
Oh, i do not see any that can help. Ok, can you make a screenshot of obsidian? Just to see a workspace.

**Comment by kitmonkman** [comment]
I’ve added a screenshot to the Dropbox folder. In the screenshot Obsidian is running various plugins. I’ve tired disabling ALL plugins, but it makes no difference to the error message.

I’m grateful to you for all your help.

V best

Kit

**Reply by snezhig** [reply]
Why there are two identical icons? Two explorers?
![изображение](https://github.com/snezhig/obsidian-front-matter-title/assets/3987716/3ff199c1-9d18-4e12-9a91-80e581a1e520)

**Comment by kitmonkman** [comment]
Hi,

I don’t know how that happened. The duplicate disappeared when I turned off files in the core plugins. When I turned it back on again, the duplicate was gone, and your plugin worked perfectly!

Thank you so much for all your help.

V best

Kit

**Reply by snezhig** [reply]
Magic. :)
